content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -372,6 +374,18 @@ implemented using the NetworkPolicy API.
 - The ability to prevent loopback or incoming host traffic (Pods cannot currently block localhost
   access, nor do they have the ability to block access from their resident node).
 
+## Recommendations for Users
+
+This section covers a set of recommended best practices to follow when using NetworkPolicies.
```

Comment:
Optional feedback, but I do recommend a change here.
The actual recommendation appears quite late in the single-item list.

I'd write:
```suggestion
## NetworkPolicy changes and impact on existing connections {#policy-change-undefined-behavior}
```

Reply:
ack I renamed it to a subsection `NetworkPolicy impact on existing connections`